cluster/vagrant/setup_kubernetes_common.sh

```diff
@@ -79,7 +79,7 @@ yum install -y \
 # Latest docker on CentOS uses systemd for cgroup management
 cat << EOT >>/etc/systemd/system/kubelet.service.d/09-kubeadm.conf
 [Service]
-Environment="KUBELET_EXTRA_ARGS=--cgroup-driver=systemd --fail-swap-on=false"
+Environment="KUBELET_EXTRA_ARGS=--cgroup-driver=systemd --fail-swap-on=false --runtime-cgroups=/systemd/system.slice --kubelet-cgroups=/systemd/system.slice"
```

Comment:
Why is this line changed? I didn't need that so far ...

Reply:
ah this will clean a little `journalctl` from messages like
```
Failed to get system container stats for "/system.slice/docker.service": failed to get cgroup stats for "/system.slice/docker.service": failed to get cgroup stats for "/system.slice/docker.service": failed to get container info for "/system.slice/docker.service": unknown container "/system.slice/docker.service"
```
it does not relate to the namespace, I just added it when I debug problems, so I can leave or remove it. Give me know what do you prefer

Reply:
sounds looks like it is good to have it.